test/integration/users.test.js

```diff
@@ -585,6 +575,49 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    afterEach(async function () {
+      await cleanDb();
+    });
+
+    it("Should return given user by id", async function () {
+      await addOrUpdate(userData[0], userId);
+      const { user: testUser } = await users.fetchUser({ userId });
```

Comment:
is this assigning data from db or directly assigning the test user to user?

Reply:
user property of the object returned by the fetchUser method to a variable called testUser.